python/common/python/time.java

```diff
@@ -89,10 +89,26 @@ public time() {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "str(object='') -> str\n"+
+                      "str(bytes_or_buffer[, encoding[, errors]]) -> str\n"+
+                      "Create a new string object from the given object. If encoding or"+
+                      "errors is specified, then the object must expose a data buffer"+
+                      "that will be decoded using the given encoding and error handler."+
+                      "Otherwise, returns the result of object.__str__() (if defined)"+
+                      "or repr(object)."+
+                      "encoding defaults to sys.getdefaultencoding().\n"+
+                      "errors defaults to 'strict'."
     )
     public static org.python.Object ctime() {
```

Comment:
This function is supposed to take an optional argument: you can pass the number of seconds:

```
>>> import time
>>> time.ctime(0)
'Thu Jan  1 01:00:00 1970'
>>> now = time.time()
>>> time.ctime(now - now % 3600)
'Mon Feb 26 15:00:00 2018'
```

The above are good candidates for tests, btw.

Reply:
I have overloaded the function `ctime()` for optional parameter, what could be the reason for the following error, 
`TypeError: ctime() takes 0 positional arguments but 1 was given`
Thanks.

Reply:
Check the docs section on Python signatures for Java-defined methods: http://voc.readthedocs.io/en/latest/reference/signatures.html